lib/openshift/role.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/project_resource'
```

Comment:
added to support `Given admin ensures "list-pod" role is deleted from the "default" project after scenario`